test_integration/geopm_test_launcher.py

```diff
@@ -283,12 +283,15 @@ def set_num_cpu(self):
             # Don't use hyper-threads.
             self._num_cpu = cpu_thread_core_socket[2] * cpu_thread_core_socket[3] - 1
 
-    def set_cpu_per_rank(self):
-        try:
-            rank_per_node = int(math.ceil(self._num_rank / self._num_node))
-            self._cpu_per_rank = int(self._num_cpu // rank_per_node)
-        except (AttributeError, TypeError):
-            pass
+    def set_cpu_per_rank(self, cpu_per_rank=None):
```

Comment:
No longer need this changes, right?

Reply:
We don't need them, but they also don't hurt.  Seems liek a feature we may need later.

Reply:
Some context for what was happening here would be useful.

Reply:
Would you prefer to delete this patch @bgeltz?  It is not needed for the feature, but without it you cannot control the threads per rank when using the test_launcher.

Reply:
I'm fine with it here, I just did not understand what it was fixing.